kitchen.dokken.yml

```diff
@@ -14,32 +14,25 @@ verifier:
   name: inspec
 
 platforms:
-  # - name: debian-8
-  #   driver:
-  #     image: dokken/debian-8
-  #     intermediate_instructions:
-  #       - RUN /usr/bin/apt-get update
-  #       - RUN /usr/bin/apt-get install sudo -y
-
   - name: debian-9
```

Comment:
Do you think we should add dev 10 in this change, or in a future one?

Reply:
If it's a simple add, then I say do it here. If it's a bit more, then do it in another PR.